content/news/2024-12-12-mixxing-up-pen-and-paper.md

```diff
@@ -0,0 +1,49 @@
+# Mixxxing up Pen and Paper
+
+> It is cold and moist. You are deep within the belly of a huge mashine surrounded by overgrowth and broken glass. You know she wants to kill you. After all: you did the same to her. She wants revenge and the only thing keeping you alive is her unstoppable desire to test.
+>
+> The sraching of plates and distant screams of broken turrets surround you. You are alone; ovserved by an all-seeing mashine. Your only chance of survival is to comply.
+>
+> So, testing. Right.
+>
+> You enter the next chamber.
+>
+> "Did you know that people with a bad conciouse get more easily startled by loud nois - " *CHOOOOOOOO*
+>
+> *She is out to kill you.*
+> 
+> The harsh red light of a lazeer illuminates the huge test chamber, when suddenly, a cube drops from the ceiling. But it is not like the others. Every Cube had a Aperture Logo on it. But this one ... a heart? Memories of betraying an old friend flash before your eyes.
+>
+> You get closer, carefully stepping over the deadly laser beam, when you hear a faint sound. It is friendly. It does not want to hurt you.
+>
+> You go closer to the cube.  
+> Is that voice coming from the ... cube?  
+> You crouch next to it.
+>
+> It hums. A small melody. Fihgting against the eary noise around you.
+
+If I had to pick a favorite moment in a video game, it would be this: a perfect exampple of how mixxing and dynamic soundtracks can not only enhancing, but telling a story outright.
+
+So: let's use this to spice up pen and paper.
+
+## Using Mixxx for Pen and Paper
+There are many different flavors of Table Top Role Playing Games like Dungeons and Dragons, Vampires the Mascarade and Fate. And there are many ways to play it. Some like it silly, some like it tectical, and I personally like it intense and immsersive.
+
+And Music is great for this.
+
+Most Storytellers will just use one of the many playlists offered by youtube or one of the many online tabletop audio apps. But that never really worked for me.
+
+One obvious reason is: Internet on Pen and Paper Cons is often bad.
+But there is one more reason.
+
+I like running campaigns in existing worlds. Alan Wake, Halo, Baldours Gate, Darksiders, Dark Souls - many games with very memorable soundtracks.
+
+So just use these Soundtracks for my campaigns?
+
+Well that is easier said than done.
+
+## How does a video game soundtrack work?
+
+Video game soundtracks are weird pieces of music, because play is unpredictable. Players will take different amounts of time tO go trough each level, so a Soundtrack has to adapt to this.
```

Comment:
```suggestion
Video game soundtracks are weird pieces of music, because play is unpredictable. Players will take different amounts of time to go trough each level, so a Soundtrack has to adapt to this.
```